golem-templates/templates/js/js-app-common/package.json

```diff
@@ -8,8 +8,8 @@
     "@golemcloud/golem-ts": "1.2.0"
   },
   "devDependencies": {
-    "@golemcloud/componentize-js": "0.10.5-golem.4",
-    "@golemcloud/jco": "1.4.4-golem.3",
+    "@bytecodealliance/jco": "^1.10.2",
```

Comment:
please use exact or pinned version (especially that we have workarounds on top of a specific version)